compare/compare.c

```diff
@@ -171,11 +171,11 @@ static int compare1(const Image16 *target, const uint16_t *grad,
 
 static int compare(const Image16 *target, const uint16_t *grad,
                    const ASS_Image *img, const char *path,
-                   double *result, int scale)
+                   double *result, int scale_x, int scale_y)
 {
-    if (scale == 1)
+    if (scale_x == 1 && scale_y == 1)
         return compare1(target, grad, img, path, result);
-    int scale2 = scale * scale;
+    int scale2 = scale_x * scale_y;
```

Comment:
It's no longer a correct name, maybe something like `area` would be better?

Reply:
Renamed to `scale_area`.